charts/atlantis/values.yaml

```diff
@@ -246,13 +246,14 @@ ingress:
 # You have to create a secret `my-ca-certificates`
 # customPem: my-ca-certificates
 
-resources:
-  requests:
-    memory: 1Gi
-    cpu: 100m
-  limits:
-    memory: 1Gi
-    cpu: 100m
+resources: {}
+# resources:
+#   requests:
+#     memory: 1Gi
+#     cpu: 100m
+#   limits:
+#     memory: 1Gi
+#     cpu: 100m
```

Comment:
This is a breaking change. Please leave it as-is and then in your own values you can set them to `{}`